src/components/widget/DatePicker.js

```diff
@@ -87,15 +87,17 @@ class DatePicker extends Component {
 
   render() {
     return (
-      <div tabIndex="-1" onKeyDown={this.handleKeydown}>
+      <div tabIndex="-1" onKeyDown={this.handleKeydown} className="datepicker">
         <Datetime
           closeOnTab={true}
           renderDay={this.renderDay}
           renderInput={this.renderInput}
           onBlur={this.handleBlur}
           onFocus={this.handleFocus}
           {...this.props}
+          open={true}
```

Comment:
@wiadev seems as if the datepickers are now always initially opened and don't close anymore.

see the following example
![image](https://user-images.githubusercontent.com/13365687/34861256-df32ca3e-f763-11e7-98e2-5ea2ac33e193.png)